uhubctl.c

```diff
@@ -232,8 +235,12 @@ static int is_rpi_5  = 0;
 static const char short_options[] =
     "l:L:n:a:p:d:r:w:s:hvefRN"
 #if defined(__gnu_linux__) || defined(__linux__)
+#if defined(LIBUSB_API_VERSION) && (LIBUSB_API_VERSION >= 0x01000107)
```

Comment:
Move this `if` outside of Linux - it should be applicable to any OS which has equivalent of USB device path (e.g. *BSD, Darwin, etc).
Let's make short name for sysdev option not `u` but `y` - I would like to keep `u` for future use e.g. `--usb`.
If block body should be simply `"y:"`.